src/main/kotlin/blackjack/view/OutputView.kt

```diff
@@ -0,0 +1,40 @@
+package blackjack.view
+
+import blackjack.domain.CardMark
+import blackjack.domain.Player
+
+object OutputView {
+    private const val SPADE_STR = "스페이드"
+    private const val HEART_STR = "하트"
+    private const val DIAMOND_STR = "다이아몬드"
+    private const val CLOVER_STR = "클로버"
+
+    fun printDefaultPlayerCards(players: List<Player>) {
+        println("${players.joinToString { it.name }}에게 2장의 카드를 나누었습니다.")
+        players.forEach { printPlayerCards(it) }
```

Comment:
이러한 구문을 함수 reference로 넘길 수 있습니다.
```suggestion
        players.forEach(::printPlayerCards)
```
lambda로 작성하는 것과 어떤 차이가 있는지 참고할 수 있는 글을 소개해드릴게요 :slightly_smiling_face:
https://proandroiddev.com/kotlin-lambda-vs-method-reference-fdbd175f6845

참고로 저는 가능한 곳에서는 함수 reference를 활용하는 것을 지향합니다. lambda로 가져온 데이터를 별도로 조작하지 않고 그대로 함수 시그니처로 넘긴다는 것을 드러낼 수 있기 때문입니다.

Reply:
코틀린에서도 가능하다는걸 리뷰 통해서 알게 되었네요! 감사합니다:)